packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -181,14 +181,7 @@ contract Deploy is Deployer {
     /// @notice Deploy a new OP Chain using an existing SuperchainConfig and ProtocolVersions
     /// @param _superchainConfigProxy Address of the existing SuperchainConfig proxy
     /// @param _protocolVersionsProxy Address of the existing ProtocolVersions proxy
-    /// @param _includeDump Whether to include a state dump after deployment
-    function runWithSuperchain(
-        address payable _superchainConfigProxy,
-        address payable _protocolVersionsProxy,
-        bool _includeDump
-    )
-        public
-    {
```

Comment:
`runWithSuperchain` isn't used anywhere in the repo, but I've chosen to keep it for now as we may want to preserve the separation between superchain and op chain deploy for deploying multiple op chains in testing.